MobileCenter/MobileCenter/Internals/Sender/MSHttpSender.m

```diff
@@ -211,6 +211,8 @@ - (void)sendCallAsync:(MSSenderCall *)call {
             @synchronized(self) {
               NSString *payload = nil;
               NSInteger statusCode = [MSSenderUtil getStatusCode:response];
+              
+              // Trying to format json for log. Don't need to log json error here.
```

Comment:
Because these serialization actions only need to [log with formatted json](https://github.com/akvelon/mobile-center-sdk-ios/blob/208d937a964c29ac70f2a340ab15791a9bf68f44/MobileCenter/MobileCenter/Internals/Sender/MSHttpSender.m#L240-L241).